osu.Game.Rulesets.Mania/UI/ManiaStage.cs

```diff
@@ -150,6 +150,41 @@ public ManiaStage(int firstColumnIndex, StageDefinition definition, ref ManiaAct
             }, true);
         }
 
+        private ISkin currentSkin;
+
+        [BackgroundDependencyLoader]
+        private void load(ISkinSource skin)
+        {
+            currentSkin = skin;
+            skin.SourceChanged += onSkinChanged;
+
+            onSkinChanged();
+        }
+
+        private void onSkinChanged()
+        {
+            foreach (var col in columnFlow)
+            {
+                if (col.Index > 0)
```

Comment:
what does index 0 mean here?

Reply:
The first column. It doesn't have spacing.

Reply:
aha, missed that